### PR TITLE
Added nikola and logo

### PR DESCRIPTION
**Pull request by ralsina**
Adding nikola to the list. I have not checked that it display correctly since it's a SVG logo it may not, please check if possible :-)

### CODE REVIEW

File: _sections/30-projects.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ These projects pledge to drop Python 2 support in or before 2020.
 - [![](assets/xonsh.png)xonsh](http://xon.sh)
 - [![](assets/astropy.png)Astropy](https://www.astropy.org/)
 - [![](assets/biopython.png)Biopython](https://biopython.org/)
+- [![](assets/nikola.png)Nikola](https://getnikola.org)
```
Comment (takluyver) [inline _sections/30-projects.md]: Did you mean https://getnikola.com/ ? The .org domain doesn't work for me.

**Comment by takluyver** [comment]
Thanks!

Do you have (or want to make) a version of the logo with a bit of margin on the background? It doesn't quite look right to me with the shapes going right to the edge. Up to you, though.

**Comment by ralsina** [comment]
Yeah, you are right, pushing it in a second.

**Comment by takluyver** [comment]
Looks good, thanks.

(For the record, I checked that @ralsina is a core contributor of nikola)

**Pull request merged**